### ISSUE TITLE
SuperPointNet_gauss2

### ISSUE THREAD

**Issue by jeethesh-pai** [eric-yyjau/pytorch-superpoint#52]
Hallo Eric-yyjau,

Thank you for your amazing work. Your codes are working without any problem. But I did not get the idea behind using SuperPointNet_gauss2 instead of original SuperPointNet given by MagicLeap. Can you please share relevant literature behind this implementation. It would be very helpful

**Comment by eric-yyjau** [comment]
Sorry for late reply. Feel free to take a look at our IROS 2020 paper:
[2007.15122] Deep Keypoint-Based Camera Pose Estimation with Geometric Constraints
https://arxiv.org/abs/2007.15122

The basic structure of the model should be the same as SuperPoint.
We add the softmax layer to propagate the loss:
https://github.com/eric-yyjau/pytorch-superpoint/blob/master/models/SuperPointNet_gauss2.py#L91

**Comment by jeethesh-pai** [comment]
Thank you for the reply. I will surely check out your paper as it seems relevant to my project.
